packages/puppeteer-crawler/tsconfig.json

```diff
@@ -1,4 +1,4 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["src/**/*"]
+  "include": ["src/**/*", "puppeteer-launcher.ts"]
```

Comment:
why? it is under `src` folder as well as exported from the root index.ts, or not?

Reply:
I accidentally moved the launcher file in VSC by mistake and it updated imports automatically 😅 Nice catch